app/assets/javascripts/rating.js

```diff
@@ -0,0 +1,32 @@
+class RatingSidebar {
+  constructor() {
```

Comment:
Hm, 🤔 I've never seen this in JS yet, are you sure this works? I know `constructor` is a thing in `.coffee`, but for JS I thought one needs to explicitly call it...?

Reply:
re: "I'm really not good at this js thing" ;)
I'll take another look at the rating js 👍